packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -127,9 +141,10 @@ class ToggleMenuDrawer extends React.Component {
 
     render() {
         const all_routes_config = getAllRoutesConfig();
-        const allowed_routes = [routes.reports, routes.account];
+        const subroutes_config = [].concat(...all_routes_config.map(i => i.routes || []));
+        const allowed_routes = [routes.reports, routes.account, routes.cashier];
```

Comment:
I think we both did the same thing here, but worth note that we need to check the cashier store before displaying `cashier_pa`, `cashier_pa_transfer` and `p2p`. I've already handled it in my PR #945 though.

Reply:
Currently if we open any cashier directly we can't see `Reports` and `Account Settings` menu in the drawer. This change will allow us to see `Reports` and `Account Settings` menu (without submenus) in the drawer even when only core gets loaded.

Reply:
Yeah understood, but I think we need to handle the subroute viewing permissions as well. 

We won't know if the user has access to `POI` or `POA` pages until a `get_account_status` is called from API. Same issue with cashier with the `pa and p2p` viewing permissions.